Tests/test_KDTree.py

```diff
@@ -51,17 +51,17 @@ def neighbor_test(nr_points, dim, bucket_size, radius):
     Returns true if the test passes.
     """
     # KD tree search
-    kdt = _CKDTree.KDTree(dim, bucket_size)
+    kdt = KDTree(dim, bucket_size)
     coords = random.random((nr_points, dim))
-    kdt.set_data(coords)
-    neighbors = kdt.neighbor_search(radius)
+    kdt.kdt.set_data(coords)
```

Comment:
This looks like an API change - why the extra ``.kdt`` here?

Reply:
Previously we made a C-level KDTree only. Now we make a KDTree at the Python level; the extra .kdt is to get to the C-level.

Reply:
So seeing this change in the test code is a reflection of the test doing something a little messy which the typical end user would not do?

Reply:
For the users, it won't matter as they would always make a KDTree at the Python level.

That said, this code is messy and should be reorganized, preferably in a new module space.

Reply:
> So seeing this change in the test code is a reflection of the test doing something a little messy which the typical end user would not do?

Yes that is correct.